### PR TITLE
DYN-5984: Update Tessellation dependency, remove InstallUpdate, mono solution

### PR DESCRIPTION
**Pull request by aparajit-pratap**
### Purpose

Update Tessellation deps, remove InstallUpdate, remove mono

TODO: 
- [x] Add deleted files to `legacy_remove_me` folder for patch installers.
- [x] Remove mono solution

### Declarations

Check these if you believe they are true

- [ ] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated
- [ ] This PR contains no files larger than 50 MB 

### Release Notes

(FILL ME IN) Brief description of the fix / enhancement. **Mandatory section** 

### Reviewers

(FILL ME IN) Reviewer 1  (If possible, assign the Reviewer for the PR)

(FILL ME IN, optional) Any additional notes to reviewers or testers.

### FYIs

(FILL ME IN, Optional) Names of anyone else you wish to be notified of

### CODE REVIEW

File: src/Libraries/Tesellation/Tessellation.csproj [hunk 1]
```diff
@@ -15,8 +15,8 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="DynamoVisualProgramming.LibG_229_0_0" Version="2.18.0.1400" />
-    <PackageReference Include="MIConvexHull" version="1.1.17.0411" CopyPDB="true" />
-	<PackageReference Include="StarMath" version="2.0.14.1114" CopyPDB="true" />
+    <PackageReference Include="MIConvexHull" version="1.1.17.411" CopyPDB="true" />
+	<PackageReference Include="StarMath" version="2.0.20.204" CopyPDB="true" />
```
Comment (aparajit-pratap) [inline src/Libraries/Tesellation/Tessellation.csproj:19]: latest, supports netstandard 2.1

Comment (mjkkirschner) [reply src/Libraries/Tesellation/Tessellation.csproj:19]: net std 2.1 does not support net.48 - see this:
Screenshot 2023-06-29 at 9 56 05 AM

only net std 1 - 2 support net framework.

File: src/Dynamo.All.Net6.sln [hunk 2]
```diff
@@ -136,6 +136,7 @@ EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DynamoSandbox", "DynamoSandbox\DynamoSandbox.csproj", "{0EFABBF0-2B81-4423-B8E2-6111FBE69B79}"
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "AssemblyRenamerCLI", "Tools\AssemblyRenamerCLI\AssemblyRenamerCLI.csproj", "{F382C3F8-C55C-4350-800A-3D13A94F8E13}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "InstallUpdate", "Tools\InstallUpdate\InstallUpdate.csproj", "{0ED387BC-17B5-49B7-9C1D-BF58A4A5CC4D}"
```
Comment (mjkkirschner) [inline src/Dynamo.All.Net6.sln:139]: can we just get rid of this project?

**Review by mjkkirschner — COMMENTED** [review]
added some questions and an issue.

File: src/Dynamo.All.Net6.sln (hunk 2 above)
Comment (aparajit-pratap) [reply src/Dynamo.All.Net6.sln:139]: Not sure, I thought you created a task for that. DynamoCrypto, etc.

Comment (aparajit-pratap) [reply src/Dynamo.All.Net6.sln:139]: Will try tracking down the issue to see if it's related.

File: src/Libraries/Tesellation/Tessellation.csproj (hunk 1 above)
Comment (aparajit-pratap) [reply src/Libraries/Tesellation/Tessellation.csproj:19]: Ok, thanks for pointing that out.

**Review by sm6srw — APPROVED** [review]
LGTM!

**Comment by sm6srw** [comment]
Actually, looks like you also need to remove the project in the mono solution.... (Do we really need to keep the mono solution around?)

**Review by sm6srw — APPROVED** [review]
LGTM!

**Comment by aparajit-pratap** [comment]
This has now passed: https://master-5.jenkins.autodesk.com/job/Dynamo/job/DynamoSelfServe/job/pullRequestValidation/9573/

**Review by mjkkirschner — CHANGES_REQUESTED** [review]
the files in legacy remove me should not be empty files - I tried that and it breaks installations because they want dlls with version numbers, certs etc.

File: src/DynamoCore/Updates/UpdateManager.cs [hunk 3]
```diff
@@ -580,6 +581,8 @@ internal sealed class UpdateManager : NotificationObject, IUpdateManager
         private static BinaryVersion productVersion;
         private IAppVersionInfo updateInfo;
         private const string OLD_DAILY_INSTALL_NAME_BASE = "DynamoDailyInstall";
+        // TODO: InstallUpdate tool has been removed. 
+        // Remove UpdateManager and and code that's dependent on it: https://jira.autodesk.com/browse/DYN-5873
```
Comment (mjkkirschner) [inline src/DynamoCore/Updates/UpdateManager.cs:585]: thanks.

File: src/Libraries/Tesellation/Tessellation.csproj [hunk 4]
```diff
@@ -15,8 +15,8 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="DynamoVisualProgramming.LibG_229_0_0" Version="2.18.0.1400" />
-    <PackageReference Include="MIConvexHull" version="1.1.17.0411" CopyPDB="true" />
-	<PackageReference Include="StarMath" version="2.0.14.1114" CopyPDB="true" />
+    <PackageReference Include="MIConvexHull" version="1.1.17.411" CopyPDB="true" />
+	<PackageReference Include="StarMath" version="2.0.17.1019" CopyPDB="true" />
```
Comment (mjkkirschner) [inline src/Libraries/Tesellation/Tessellation.csproj:19]: any license changes? Have you updated version info in the about box, license txt and third party wiki?

File: extern/legacy_remove_me/.gitignore [hunk 5]
```diff
@@ -1,3 +1,5 @@
 # override specific package dlls#
 ###################
 !bin/
+!*.exe
+!*.pdb
```
Comment (mjkkirschner) [inline extern/legacy_remove_me/.gitignore:5]: did we really deliver .pdbs to our integrators?

Comment (aparajit-pratap) [reply extern/legacy_remove_me/.gitignore:5]: The bin-diff job highlighted the deletion of the pdb file so I thought it was safer to add it back as the job is run against release builds if I'm not mistaken.

File: src/Libraries/Tesellation/Tessellation.csproj (hunk 4 above)
Comment (aparajit-pratap) [reply src/Libraries/Tesellation/Tessellation.csproj:19]: Done

**Comment by aparajit-pratap** [comment]
> the files in legacy remove me should not be empty files - I tried that and it breaks installations because they want dlls with version numbers, certs etc.

So what version number should it be - the version that was shipped at GL?

**Comment by mjkkirschner** [comment]
> 

I don't think it matters, we're not going to load them, so I would make them the versions from the previous release, thats clearer I think.

**Review by mjkkirschner — APPROVED** [review]

**Pull request merged**